source/_patterns/00-protons/printing/_bootstrap-printing.scss

```diff
@@ -0,0 +1,45 @@
+// Bootstrap's CSS-generating output
+//
+// This is the exact order within:
+//   ~bootstrap/scss/bootstrap
+// Override all Bootstrap variables at:
+//   00-protons/non-printing/_bootstrap-overrides.scss
+//
+// Feel free to move each import below to its respective atom/molecule/organism
+// _component.scss file. All are included here for simplicity.
+
+//@import "~bootstrap/scss/functions"; //_bootstrap-overrides.scss
+//@import "~bootstrap/scss/variables"; //_bootstrap-overrides.scss
+//@import "~bootstrap/scss/mixins"; //_bootstrap-overrides.scss
+@import '~bootstrap/scss/root';
+@import '~bootstrap/scss/reboot';
+@import '~bootstrap/scss/type';
+@import '~bootstrap/scss/images';
+@import '~bootstrap/scss/code';
+@import '~bootstrap/scss/grid';
```

Comment:
i see the note about including everything here for simplicity, but it seems like a solid vote for moving grids from atoms to protons as a "printing" config or demo

Reply:
Our grids actually have usable twig patterns which are used a ton. There is no includable twig that lives in protons.

Reply:
that is true, but it feels a little weird to me still that most of our protons demos rely on the grid atom (or the card sass, for colors). its a semantic point, but a grid system seems like part of project config. not a blocker here

Reply:
Ah, good point: a proton demo that needs an atom ... Hrm...

Reply:
Here are my stated reasons for keeping grid an atom:

1. Protons are variable/styling/logical generating **only**. The *first* concept in our take on Atomic Design that actually `{% include|embed|extends %}`'s are atoms. 
1. Note the structure of protons vs the other categories: 
![image](https://user-images.githubusercontent.com/295127/46625380-a1c3fe80-cae8-11e8-939d-edd7e69dd878.png). Protons includes no other standalone components, on purpose.
1. Protons only uses grid *as part of its demos*.  Grid is not being used as an actual inclusion in another component, just for a demo purpose. We should not conflate inclusion in a demo for inclusion in a component!

Reply:
This reasoning makes sense to me and I'm good with it! 

May I offer as an alternative:
1) Remove use of the grid pattern in the proton demos and leave un-styled or (preferably);
2) Write Out default bootstrap in proton demos, e.g., manually write out `<div class="container">`. 

My reason being is that I feel we should be able to reliably delete components top-down without any breakages or webpack errors. e.g, Delete components from pages => atoms without the user having to diagnose what broke their build.

Reply:
I don't hate this.

Reply:
If you like, I can open a separate issue for this this and tackle it after this current Bootstrap PR is merged?

Reply:
Naw, I'll get this in on this PR. This will be good code to write today while I should be paying attention in meetings.

Reply:
Done. https://github.com/phase2/particle/pull/411/commits/d69162e375393c146c1223c667a48a70375419b0